src/Microsoft.DotNet.Build.Tasks/Targets/versioning.targets

```diff
@@ -0,0 +1,43 @@
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="GetDailyBuildNumber" 
+    AssemblyFile="$(ToolsDir)Microsoft.DotNet.Build.Tasks.dll"
+    Condition="Exists('$(ToolsDir)Microsoft.DotNet.Build.Tasks.dll')" />
+
+  <Target Name="GetDailyBuildNumber"
+    BeforeTargets="Build"
+    Condition="'$(DailyBuildNumGenEnabled)' == 'true'"
+    >
+    <GetDailyBuildNumber
```

Comment:
Just for clarity I think it would be worth calling this Target GetNextDailyBuildNumber.